src/blocks/tweet.js

```diff
@@ -100,5 +100,12 @@ module.exports = Block.extend({
   onDrop: function(transferData){
     var url = transferData.getData('text/plain');
     this.handleTwitterDropPaste(url);
+  },
+
+  asClipboardHTML: function() {
+    var data = this.getBlockData();
+    if (!data.status_url) return;
+
+    return `<p>${data.status_url}</p>`;
```

Comment:
Isn't there more data that could be used here to include the tweet text as well?

Reply:
I don't think we have access to the tweet text.

Reply:
I'm going to do some further testing with word and google docs to see what the best way of include complex blocks. I was talking with Callum and he gave some ideas I'm going to look at.